tests/python/gaia-ui-tests/gaiatest/tests/graphics/RTL/test_settings_NC_RTL.py

```diff
@@ -7,43 +7,196 @@
 from gaiatest.apps.settings.app import Settings
 
 
-class TestSettingsRTL(GaiaImageCompareTestCase):
+class TestSettingsRTLNC(GaiaImageCompareTestCase):
     def test_settings_app(self):
 
         settings = Settings(self.marionette)
         settings.launch()
 
-        # opening each subpage in Settings menu.  Privacy Control is not opened, because it is a separate app
-        #some subpages have their own subpages, and they need to be opened as well.
-        settings.open_wifi()
-        self.take_screenshot('wifi')
-        settings.return_to_prev_menu(settings.screen_element)
+        # opening each subpage in Settings menu.
+        # some subpages have their own subpages, and they need to be opened as well.
+        ##################### Wi-Fi and its subpages ######################
+        wifi_page = settings.open_wifi()
+        Wait(self.marionette).until(lambda m: wifi_page.is_wifi_enabled is True)
+        
+        self.take_screenshot('wifi-enabled')
+        wifi_page.tap_connect_with_wps()
+        self.take_screenshot('wifi-wps')
+        settings.return_to_prev_menu(wifi_page.screen_element, wifi_page.wps_screen_element)
+        self.marionette.execute_script('arguments[0].scrollIntoView(false);', [wifi_page.manage_network_button])
+        wifi_page.tap_manage_networks()
+        wifi_page.tap_join_hidden_network()
+        wifi_page.tap_security_selector()
+        self.take_screenshot('wifi-securityType')
+        wifi_page.tap_security_ok()
+        self.take_screenshot('wifi-joinHidden')
+        settings.return_to_prev_menu(wifi_page.manage_network_screen_element, 
+                                     wifi_page.join_hidden_network_screen_element)
+        self.take_screenshot('wifi-manageNetwork')
+        settings.return_to_prev_menu(wifi_page.screen_element, wifi_page.manage_network_screen_element)
+        wifi_page.tap_manage_certs()
+        wifi_page.tap_import_certs()
+        self.take_screenshot('wifi-importCert')
+        settings.return_to_prev_menu(wifi_page.manage_certs_screen_element, wifi_page.select_certs_screen_element)
+        self.take_screenshot('wifi-manageCert')
+        settings.return_to_prev_menu(wifi_page.screen_element, wifi_page.manage_certs_screen_element)
+        wifi_page.disable_wifi()
+        self.take_screenshot('wifi-disabled')
+        settings.return_to_prev_menu(settings.screen_element, wifi_page.screen_element)
 
-        settings.open_sim_manager()
+        #################### Sim manager and its subpages ######################
+        sim_page = settings.open_sim_manager()
         self.take_screenshot('sim_manager')
-        settings.return_to_prev_menu(settings.screen_element)
+        GaiaImageCompareTestCase.scroll(self.marionette, 'down', sim_page.screen_element.size['height'],
+                                        screen=sim_page.screen_element)
```

Comment:
I think you could also indent 4 characters past GaiaImageCompareTestCase.
I'm not sure we have a convention here, I think I've seen this done in both ways.